deployment/aws.sh

```diff
@@ -14,7 +14,7 @@ env
 GIT_BRANCH=${GIT_BRANCH:-develop}
 
 /usr/bin/apt-get -q -y install htop
-/usr/bin/apt-get -q -y install awscli jq postgresql-client-9.3
+/usr/bin/apt-get -q -y install awscli jq postgresql-client-10
```

Comment:
I don't think this will work because the apt repo that provides postgresql-client-10 is not configured at this point.

Reply:
@hackdna You're right, but after looking into it I think the only reason we had to do the `postgresql-client` install in `aws.sh` was due to the [`ensure-postgresql-database` script](https://github.com/refinery-platform/refinery-platform/pull/3061/files#diff-69a92f04c759417a5a4dc7c0821e122cL89).

Since that was removed in #3061, I don't think we need the `postgresql-client` in this script anymore.

Reply:
OK then remove it and check if the stack still comes up?

Reply:
Great. This should probably be the last PR merged before the release to make bringing up stacks using RDS snapshots easier in the meantime.